server/src/connectionHandlers/onDefinition.ts

```diff
@@ -104,6 +104,28 @@ export function onDefinitionHandler (textDocumentPositionParams: TextDocumentPos
       })
       return definitions
     }
+    // Overrides
+    if (analyzer.isOverride(documentUri, position.line, position.character)) {
+      const overrideFile = bitBakeProjectScannerClient.bitbakeScanResult._confFiles.find((confFile) => {
```

Comment:
I think you can just use lastScanResult (recipe) here? Are there instances were the .conf was only in the bitbakeScan?

Reply:
Now it prioritizes the paths found in the recipe scan results. check af6cb9ad185f9f44d219666bface48d3ee03ba52.
If the recipe scan was never executed, there will be no last scan result for the target recipe. If the initial project scan got the file we want, we can use it which is nice.